.github/workflows/cli-validation.yml

```diff
@@ -1,4 +1,4 @@
-name: fern
+name: CLI Validation
```

Comment:
this repo checks the fern definitions in the fern repo (hosted at `/fern`)

Reply:
revised to: `name: Validate Fern Definitions` and `definitions-validation.yml`